### PR TITLE
feature/378 allow user to go back and generate new spreadsheet

### PR DESCRIPTION
**Pull request by Ashalynd**
New PR due to renaming of the branch.

replaces https://github.com/StarterSquad/Optima/pull/206

https://trello.com/c/GqxNj7id/378-5-allow-user-to-go-back-and-generate-new-spreadsheet

(check the card for the description of expected functionality)

### CODE REVIEW

**Comment by Ashalynd** [comment]
don't merge this yet, needs some changes first

**Comment by Ashalynd** [comment]
@dmitryevseev or @nikgraf please also have a look on this one.

File: client/source/js/modules/project/create-ctrl.js [hunk 1]
```diff
@@ -2,19 +2,58 @@ define(['./module', 'angular', 'underscore'], function (module, angular, _) {
...
+    $scope.editParams = {
+      is_edit: false
+    };
+    $scope.project_info = info;
 
     var availableParameters = parametersResponse.data.params;
     var availableDefaults = defaultsResponse.data;
-
+        
+    $scope.submit = "Create project & Optima template";
```
Comment (nikgraf) [inline client/source/js/modules/project/create-ctrl.js]: What's the benefit of moving text to the controller code?

File: client/source/js/modules/project/create-ctrl.js [hunk 2]
```diff
@@ -2,19 +2,58 @@ define(['./module', 'angular', 'underscore'], function (module, angular, _) {
...
-    $timeout, activeProject, parametersResponse, defaultsResponse, UserManager) {
+    $timeout, activeProject, parametersResponse, defaultsResponse, info, UserManager, modalService) {
 
     $scope.projectParams = {
       name: ''
     };
+    $scope.editParams = {
+      is_edit: false
+    };
+    $scope.project_info = info;
```
Comment (nikgraf) [inline client/source/js/modules/project/create-ctrl.js]: Would be nice to stick to camelCase :smile:

File: client/source/js/modules/project/create-ctrl.js (hunk 1 above)
Comment (Ashalynd) [inline client/source/js/modules/project/create-ctrl.js]: true, not much.

**Pull request merged**